docs/manage/fleet/provision/setup.md

```diff
@@ -477,7 +483,7 @@ For a guide you can give to end users for setting up their machine, see [Setup m
 
 1. When you power on the machine that has `viam-agent` installed and `agent-provisioning` configured, `viam-agent` creates a WiFi hotspot.
 
-   - The [`agent-provisioning` configuration](/manage/fleet/provision/setup/#configure-agent-provisioning) is at <file>/etc/viam-provisioning.json</file> on your machine.
+   - The [`agent-provisioning` configuration](/manage/fleet/provision/setup/#configure-agent-provisioning) is at <file>/etc/viam-defaults.json</file> on your machine.
```

Comment:
This isn't just the "agent-provisioning" config. As noted above, there is no longer anything specifically called "agent-provisioning." The defaults file is also not JUST for provisioning. It can be used to set ANY configuration value for agent to a preferred/custom default. Not sure if documentation explaining that belongs on this page or elsewhere though.